src/NServiceBus.Core/Persistence/Learning/SagaPersister/LearningSagaPersistence.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace NServiceBus.Features
+{
+    using System;
+    using System.IO;
+    using NServiceBus.Sagas;
+
+    class LearningSagaPersistence : Feature
+    {
+        internal LearningSagaPersistence()
+        {
+            DependsOn<Sagas>();
+            Defaults(s => s.Set<ISagaIdGenerator>(new LearningSagaIdGenerator()));
+            Defaults(s => s.SetDefault(StorageLocationKey, Path.Combine(AppDomain.CurrentDomain.BaseDirectory, ".sagas")));
```

Comment:
Instead of just ".sagas" as the default, would it make sense to be something like ".learningpersistence/sagas" instead to be able scope the data better? It also mimics the learning transport a better as well.